### PR TITLE
Add cross-cloud-region-hybrid + hardhat

### PR DESCRIPTION
**Pull request by akegaviar**
Added cross-cloud, cross-region, hybrid networks.
Also added Hardhat

### CODE REVIEW

**Review by neewy — APPROVED** [review]

File: docs/glossary/hybrid.md [hunk 1]
```diff
@@ -1,15 +1,18 @@
...
-    content: A hybrid node or network deployment is the combination of nodes deployed in the cloud and on user premises.
+    content: A hybrid network is the combination of nodes deployed in the Chainstack-provided cloud providers and private hosting.
   - name: keywords
     content: hybrid blockchain deployment on-prem cloud
 ---
 
 # Hybrid
 
-A hybrid deployment means that some nodes of the same network are [on-premises](/glossary/on-premises) while others are on the public [cloud](/glossary/cloud).
+Deployment of nodes of the same network in Chainstack-provided cloud hosting providers and private hosting.
```
Comment (easeev) [inline docs/glossary/hybrid.md:11]: Technically can be also a combination of multiple private hostings

Comment (akegaviar) [reply docs/glossary/hybrid.md:11]: Fixed.

File: docs/glossary/hybrid.md [hunk 2]
```diff
@@ -1,15 +1,18 @@
...
-    content: A hybrid node or network deployment is the combination of nodes deployed in the cloud and on user premises.
+    content: A hybrid network is the combination of nodes deployed in the Chainstack-provided cloud providers and private hosting.
   - name: keywords
     content: hybrid blockchain deployment on-prem cloud
 ---
 
 # Hybrid
 
-A hybrid deployment means that some nodes of the same network are [on-premises](/glossary/on-premises) while others are on the public [cloud](/glossary/cloud).
+Deployment of nodes of the same network in Chainstack-provided cloud hosting providers and private hosting, or a combination of private hostings.
```
Comment (easeev) [inline docs/glossary/hybrid.md:11]: I think can be simplified a bit

```suggestion
Deployment of nodes of the same network in any combination of Chainstack-provided cloud hosting providers and private hosting providers.
```

**Comment by easeev** [comment]
Do we have a connection between "integration" and "private hosting"? I suggest adding it if not, otherwise it might be detached/confusing

**Comment by akegaviar** [comment]
> Do we have a connection between "integration" and "private hosting"? 

We do in multiple places: eg in `/platform/deploy-a-consortium-network`, `/platform/supported-hosting-options`, `/glossary/hybrid`.

**Review by neewy — APPROVED** [review]

**Review by easeev — APPROVED** [review]
LGTM

**Pull request merged**